packages/fiori-freestyle/templates/basic/add/webapp/Component.js

```diff
@@ -1,7 +1,7 @@
 sap.ui.define([
 		"<%=app.baseComponent%>",
 		"sap/ui/Device",
-		"<%=app.id.replace('.', '/')%>/model/models"
+		"<%=app.id.replace(/\./g, '/')%>/model/models"
```

Comment:
Oha ... that is bad bug

Reply:
@vobu here is a report of the issue you found: https://github.com/SAP/open-ux-tools/issues/70. Would you mind fixing it in a separate PR, so that we can keep it separate?

Reply:
will do this morning

Reply:
&rarr; https://github.com/SAP/open-ux-tools/pull/77